### PR TITLE
feat(front-end): Adds date dividers to visits

### PR DESCRIPTION
**Pull request by nchapman**
This should be merged after #265, #266, and #267.

* * *

Date dividers for visits with really basic relative dates (Today and Yesterday).

@6a68 @pdehaan r?

### CODE REVIEW

**Comment by nchapman** [comment]
It's worth pointing out that the date headers are unstyled. More fun for @johngruen.

File: app/styles/modules/_global_header.scss [hunk 1]
```diff
@@ -10,16 +10,43 @@ header {
...
+
+    img {
+      border-radius: 50%;
+      height: 36px;
+      width: 36px;
+    }
+
+    nav {
+      background: #fff;
+      border: 1px solid #eee;
```
Comment (pdehaan) [inline app/styles/modules/_global_header.scss]: We should use `#ffffff` and `#eeeeee` for consistency (although I think @johngruen may eventually want to refactor these out into variables or something so we can track color use a bit easier).

**Comment by pdehaan** [comment]
Not sure if it is this commit, one of the related PRs, or bad config on my end, but I'm not seeing the page update after I click a "Delete" link. I see the DELETE verb happening in my console, but the page doesn't update. THEN, if I try and foolishly click the Delete button again (LIKE A FOOL), then it spews the following error in my console:

```
backbone.js:1594 Uncaught Error: A "url" property or function must be specified
backbone.js:1594 urlError
backbone.js:537 _.extend.url
underscore.js:1245 _.result
backbone.js:1146 Backbone.sync
backbone.js:286 _.extend.sync
backbone.js:525 _.extend.destroy
item.js:40 BaseView.extend.destroyModel
jquery.js:4430 jQuery.event.dispatch
jquery.js:4116 jQuery.event.add.elemData.handle
```

**Comment by nchapman** [comment]
@pdehaan yeah I realized I broke delete in #267. I'll get that fixed.

**Comment by 6a68** [comment]
yisssssssss! code looks fine, merging

**Pull request merged**